### PR TITLE
Changed print calls to Logger

### PR DESCRIPTION
**Pull request by meriouma**

### CODE REVIEW

**Comment by Pacane** [comment]
LGTM

File: shared/lib/loglevel_parser.dart [hunk 1]
```diff
@@ -0,0 +1,32 @@
+library loglevel_parser;
```
Comment (christiangoudreau) [inline shared/lib/loglevel_parser.dart]: License header?

Comment (Pacane) [inline shared/lib/loglevel_parser.dart]: Maybe we should add a story to add license headers everywhere it's needed.

Comment (Pacane) [inline shared/lib/loglevel_parser.dart]: Added this to the board https://tree.taiga.io/project/pacane-pokerplanning/us/35

**Pull request merged**